### PR TITLE
ID localization for access-application-cluster list-image

### PR DESCRIPTION
**Pull request by ariscahyadi**
This PR translates `/docs/tasks/access-application-cluster/list-all-running-container-images/` as mentioned #22296

### CODE REVIEW

File: content/id/docs/tasks/access-application-cluster/list-all-running-container-images.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+
+
+<!-- steps -->
+
+Dalam latihan ini kamu akan menggunakan kubectl untuk mengambil semua Pod yang
+berjalan dalam sebuah klaster, dan memformat keluaran untuk melihat daftar
```
Comment (phanama) [inline content/id/docs/tasks/access-application-cluster/list-all-running-container-images.md:24]:
```suggestion
berjalan dalam sebuah klaster, dan mengubah format keluarannya untuk melihat daftar
```

File: content/id/docs/tasks/access-application-cluster/list-all-running-container-images.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+Container untuk masing-masing Pod.
+
+## Membuat daftar semua _image_ Container pada semua Namespace
+
+- Silakan ambil semua Pod dalam Namespace dengan menggunakan perintah `kubectl get pods --all-namespaces`
+- Silakan format keluarannya agar hanya menyertakan daftar nama _image_ dari Container
+  dengan menggunakan perintah `-o jsonpath={..image}`.  Perintah ini akan memparsing _field_
```
Comment (phanama) [inline content/id/docs/tasks/access-application-cluster/list-all-running-container-images.md:31]:
```suggestion
  dengan menggunakan perintah `-o jsonpath={..image}`.  Perintah ini akan mem-_parsing_ _field_
```

File: content/id/docs/tasks/access-application-cluster/list-all-running-container-images.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+  `image` dari keluaran json yang dihasilkan.
+  - Silakan lihat [referensi jsonpath](/docs/user-guide/jsonpath/)
+    untuk informasi lebih lanjut tentang cara menggunakan jsonpath.
+- Silakan format keluaran dengan menggunakan utilitas standar: `tr`, `sort`, `uniq`
```
Comment (phanama) [inline content/id/docs/tasks/access-application-cluster/list-all-running-container-images.md:35]:
```suggestion
- Silakan format keluaran dengan menggunakan peralatan standar: `tr`, `sort`, `uniq`
```

File: content/id/docs/tasks/access-application-cluster/list-all-running-container-images.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+tr -s '[[:space:]]' '\n' |\
+sort |\
+uniq -c
+```
+
+Perintah di atas secara berulang akan mengembalikan semua _field_ bernama `image`
+dari semua poin yang dikembalikan.
+
+Sebagai pilihan, dimungkinkan juga untuk menggunakan jalur (_path_) absolut ke _field image_
+dalam Pod. Hal ini memastikan _field_ yang diambil benar
```
Comment (phanama) [inline content/id/docs/tasks/access-application-cluster/list-all-running-container-images.md:51]:
```suggestion
di dalam Pod. Hal ini memastikan _field_ yang diambil benar
```

File: content/id/docs/tasks/access-application-cluster/list-all-running-container-images.md [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+bahkan ketika nama _field_ tersebut diulangi,
+misalnya banyak _field_ disebut dengan `name` dalam poin yang diberikan:
```
Comment (phanama) [inline content/id/docs/tasks/access-application-cluster/list-all-running-container-images.md:53]:
```suggestion
misalnya banyak _field_ disebut dengan `name` dalam sebuah poin yang diberikan:
```

File: content/id/docs/tasks/access-application-cluster/list-all-running-container-images.md [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+
+- `.items[*]`: untuk setiap nilai yang dihasilkan
+- `.spec`: untuk mendapatkan spesifikasi
+- `.containers[*]`: untuk setiap Container
+- `.image`: untuk mendapatkan _image_
+
+{{< note >}}
+Pada saat mengambil sebuah Pod berdasarkan namanya, misalnya `kubectl get pod nginx`,
+bagian `.items[*]` dari jalur harus dihilangkan karena hanya akan menghasilkan sebuah Pod
+sebagai keluarannya, bukannya daftar dari semua poin.
```
Comment (phanama) [inline content/id/docs/tasks/access-application-cluster/list-all-running-container-images.md:69]:
```suggestion
sebagai keluarannya, bukan daftar dari semua Pod.
```

File: content/id/docs/tasks/access-application-cluster/list-all-running-container-images.md [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+
+Format dapat dikontrol lebih lanjut dengan menggunakan operasi `range` untuk 
+melakukan iterasi untuk setiap elemen secara individual.
+
+```sh
+kubectl get pods --all-namespaces -o=jsonpath='{range .items[*]}{"\n"}{.metadata.name}{":\t"}{range .spec.containers[*]}{.image}{", "}{end}{end}' |\
+sort
+```
+
+## Membuat daftar filter _image_ berdasarkan label dari Pod
```
Comment (phanama) [inline content/id/docs/tasks/access-application-cluster/list-all-running-container-images.md:83]:
```suggestion
## Membuat daftar _image_ yang difilter berdasarkan label dari Pod
```

File: content/id/docs/tasks/access-application-cluster/list-all-running-container-images.md [hunk 8]
```diff
@@ -0,0 +1,129 @@
...
+## Membuat daftar _image_ Container yang difilter berdasarkan Namespace Pod
+
+Untuk hanya menargetkan Pod pada Namespace tertentu, gunakankan tanda Namespace. Filter
+dibawah ini hanya menyaring Pod pada Namespace `kube-system`.
+
+```sh
+kubectl get pods --namespace kube-system -o jsonpath="{..image}"
+```
+
+## Membuat daftar _image_ Container dengan menggunakan go-template bukan jsonpath
```
Comment (phanama) [inline content/id/docs/tasks/access-application-cluster/list-all-running-container-images.md:101]:
```suggestion
## Membuat daftar _image_ Container dengan menggunakan go-template sebagai alternatif jsonpath
```

File: content/id/docs/tasks/access-application-cluster/list-all-running-container-images.md [hunk 9]
```diff
@@ -0,0 +1,129 @@
...
+
+Sebagai alternatif untuk jsonpath, kubectl mendukung penggunaan [go-templates](https://golang.org/pkg/text/template/)
+untuk menformat keluaran seperti berikut:
```
Comment (phanama) [inline content/id/docs/tasks/access-application-cluster/list-all-running-container-images.md:104]:
```suggestion
untuk memformat keluaran seperti berikut:
```

File: content/id/docs/tasks/access-application-cluster/list-all-running-container-images.md [hunk 10]
```diff
@@ -0,0 +1,129 @@
...
+
+
+
+## {{% heading "whatsnext" %}}
+
+
+### Referensi
+
+* [Jsonpath](/docs/user-guide/jsonpath/) panduan referensi
+* [Go template](https://golang.org/pkg/text/template/) panduan reference
```
Comment (phanama) [inline content/id/docs/tasks/access-application-cluster/list-all-running-container-images.md:125]:
```suggestion
* Panduan [Go template](https://golang.org/pkg/text/template/).
```

File: content/id/docs/tasks/access-application-cluster/list-all-running-container-images.md [hunk 11]
```diff
@@ -0,0 +1,129 @@
...
+<!-- discussion -->
+
+
+
+## {{% heading "whatsnext" %}}
+
+
+### Referensi
+
+* [Jsonpath](/docs/user-guide/jsonpath/) panduan referensi
```
Comment (phanama) [inline content/id/docs/tasks/access-application-cluster/list-all-running-container-images.md:124]:
```suggestion
* Panduan [Jsonpath](/docs/user-guide/jsonpath/).
```

File: content/id/docs/tasks/access-application-cluster/list-all-running-container-images.md (hunk 4 above)
Comment (ariscahyadi) [reply content/id/docs/tasks/access-application-cluster/list-all-running-container-images.md:51]: "pada" is better.

**Comment by ariscahyadi** [comment]
> Thanks for the translation! I've left some reviews to improve clarity @ariscahyadi

Thanks, @phanama for your review. I have addressed them all. Please check!

File: content/id/docs/tasks/access-application-cluster/list-all-running-container-images.md [hunk 12]
```diff
@@ -0,0 +1,129 @@
...
+Untuk hanya menargetkan Pod pada Namespace tertentu, gunakankan tanda Namespace. Filter
+dibawah ini hanya menyaring Pod pada Namespace `kube-system`.
+
+```sh
+kubectl get pods --namespace kube-system -o jsonpath="{..image}"
+```
+
+## Membuat daftar _image_ Container dengan menggunakan go-template sebagai alternatif dari jsonpath
+
+Sebagai alternatif untuk jsonpath, kubectl mendukung penggunaan [go-templates](https://golang.org/pkg/text/template/)
```
Comment (sftim) [inline content/id/docs/tasks/access-application-cluster/list-all-running-container-images.md:103]: Possibly:
```suggestion
Sebagai alternatif untuk `jsonpath`, kubectl mendukung penggunaan [go template-template](https://golang.org/pkg/text/template/)
```

What do you think? (It's also fine to use your original wording if you prefer that).
or
```suggestion
Sebagai alternatif untuk `jsonpath`, kubectl mendukung penggunaan [go-template](https://golang.org/pkg/text/template/)
```

**Pull request merged**